docs/usage/rust.md

```diff
@@ -19,11 +19,18 @@ Renovate supports upgrading dependencies in `Cargo.toml` files and their accompa
 
 Renovate updates Rust crates by default.
 
-## Private crate registries and private Git dependencies
+## Private git-based crate registries / private Git dependencies
 
 If any dependencies are hosted in private Git repositories, [Git Authentication for cargo](https://doc.rust-lang.org/cargo/appendix/git-authentication.html) must be set up.
 
 If any dependencies are hosted on private crate registries (i.e., not on `crates.io`), then credentials should be set up in such a way that the Git command-line is able to clone the registry index.
 Third-party crate registries usually provide instructions to achieve this.
 
 Both of these are currently only possible when running Renovate self-hosted.
+
+## Private sparse index registries
+
+If dependencies are available via a sparse index, then a `registryAlias` can be
+used to have Renovate accesss the dependencies via the sparse protocol. Use
+`HostRules` to provide authentication details if needed. Crates-io index access
+will still be via Renovates shared clone of the public git index.
```

Comment:
```suggestion
If dependencies are available via a "sparse index", you may use `registryAliases` to let Renovate accesses the dependencies via the `sparse` protocol.
If needed, set the authentication details with the `hostRules` config option.
`crates.io` index access will still be via Renovates shared clone of the public Git index.
```

Rewrite:
  - Use `you` to draw the reader into the text
  - Put "sparse index" between quotes, to highlight the term
  - Use simple words and phrases
  - One sentence per line
  - Use `crates.io` to match line 26 of this file
  
Grammar/typos:

- The `hostRules` config option name starts with a normal case latter
- The config option is `registryAliases` not `registryAlias`
- Git is a proper noun, so must be capitalized